Bottom: fb5c50d700cf9e8f2bb0c4c4b3f904538806581e
Top:    64c551789aa853d05acd75b729034cd785582eac
Author: Sean Hefty <sean.hefty@intel.com>
Date:   2012-05-30 09:25:04 -0700

Refresh of rstream-opt

---

diff --git a/examples/rstream.c b/examples/rstream.c
index c77eaf3..48e2771 100644
--- a/examples/rstream.c
+++ b/examples/rstream.c
@@ -321,11 +321,11 @@ static int sync_test(void)
 {
 	int ret;
 
-	ret = dst_addr ? send_xfer(8) : recv_xfer(8);
+	ret = dst_addr ? send_xfer(16) : recv_xfer(16);
 	if (ret)
 		return ret;
 
-	return dst_addr ? recv_xfer(8) : send_xfer(8);
+	return dst_addr ? recv_xfer(16) : send_xfer(16);
 }
 
 static int run_test(void)
